var/spack/repos/builtin/packages/fastjet/package.py

```diff
@@ -59,6 +59,7 @@ class Fastjet(AutotoolsPackage):
 
     variant("shared", default=True, description="Builds a shared version of the library")
     variant("auto-ptr", default=False, description="Use auto_ptr")
+    variant("limited-thread-safety", default=True, description="Enables limited thread safety")
```

Comment:
Is this limited compared to full thread safety or limited compared to no thread safety at all? The description doesn't make that clear.

Reply:
There also appears to be a `--thread-safety` option too which would be useful to expose here too. Can this be a multi value with none, limited, full?

Reply:
`--enable-thread-safety` enables full thread safety and `--enable-limited-thread-safety` enables limited thread safety compared to full thread safety.